.evergreen.yml

```diff
@@ -241,32 +198,34 @@ functions:
       params:
         working_dir: src
         shell: bash
+        env:
+          <<: *compass-env
+          # TODO: npm_loglevel, debug, compass_distribution
         script: |
           set -e
-          source ~/compass_env.sh
+          eval $(node scripts/print-compass-env.js);
 
+          # TODO: move
           cat <<EOF_BUILD_SH > ~/compass_package.sh
           #! /usr/bin/env bash
+          # TODO: why?
           cd $(pwd)
 
           # Load environment variables
-          source ~/compass_env.sh
+          # TODO: won't be needed once it is a script
+          eval $(node scripts/print-compass-env.js);
 
           # For debugging any problems when the notary service fails.
           # @see https://github.com/mongodb-js/notary-service-client For details on what notary service is.
           export DEBUG=mongodb-notary*
 
           # Required to sign release assets
-          export NOTARY_URL="http://notary-service.build.10gen.cc:5000"
-          export NOTARY_AUTH_TOKEN="${signing_auth_token}"
-          export NOTARY_SIGNING_KEY="${signing_key_name}"
-          export NOTARY_SIGNING_COMMENT="Evergreen project mongodb/compass ${revision} - ${build_variant} - ${branch_name}"
 
           if [ `uname` == Darwin ]; then
             # We do print the environment during the build process in a few places
             # but we probably should stop doing that. After that, we can go back
             # to using plain APPLE_USERNAME and APPLE_PASSWORD environment variables.
-            echo '{"appleUsername": "${apple_username}", "applePassword": "${apple_password}"}' > /tmp/compass-apple-cred.json
+            echo $APPLE_CREDENTIALS > /tmp/compass-apple-cred.json
```

Comment:
Must see if we still need this workaround - maybe we can just send the apple credentials as an env var without writing to file first? It wasn't preventing the secret from being leaked in logs anyway because the `echo '{"appleUsername": "${apple_username}", "applePassword": "${apple_password}"}' > /tmp/compass-apple-cred.json` line with expansions expanded gets printed to the logs.